src/e2e/simulator.test.ts

```diff
@@ -5,16 +5,23 @@
  */
 import { test } from "./app-test-fixtures.js";
 
-const basicTest = "from microbit import *\ndisplay.show(Image.NO)";
+const basicTest = `from microbit import *
+display.show(Image.NO)`;
 
-const buttonTest =
-  "from microbit import *\nwhile True:\nif button_a.was_pressed():\ndisplay.show(Image.NO)";
+const buttonTest = `from microbit import *
```

Comment:
I've corrected the indentation in these tests which surprised me when I saw them in failure screenshots (for reasons unrelated to the indentation).